HISTORY.rst

```diff
@@ -7,6 +7,8 @@ Release History
 +++++++++++++++
 
 **IoT DPS changes**
+* DPS now supports auto resource and policy discovery. Resource group is no longer a
```

Comment:
There is an explanation for resource discovery but we should also have a short explanation for what policy discovery means.